pkg/daemon/server/handle.go

```diff
@@ -310,18 +310,37 @@ func (cdh *cniDaemonHandler) errorWrapper(err error, status int, resp *restful.R
 	})
 }
 
+func (cdh *cniDaemonHandler) listAvailableIPInstanceOfPod(podUID, podNamespace string) ([]*networkingv1.IPInstance, error) {
+	ipInstanceList := &networkingv1.IPInstanceList{}
+	if err := cdh.mgrClient.List(context.TODO(), ipInstanceList, client.InNamespace(podNamespace), client.MatchingLabels{
+		constants.LabelNode:   cdh.config.NodeName,
+		constants.LabelPodUID: podUID,
+	}); err != nil {
+		return nil, err
+	}
+
+	var availableIPInstances []*networkingv1.IPInstance
+	for _, ip := range ipInstanceList.Items {
+		if ip.DeletionTimestamp.IsZero() {
+			availableIPInstances = append(availableIPInstances, &ip)
```

Comment:
it's not right to address of a iterator.

Reply:
fixed